data/json/items/tool/tailoring.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "id": "awl_bone",
```

Comment:
You should also use copy-from for these awls.

Reply:
From each other, or something else?

Reply:
From eachother